doc/sphinx/language/core/definitions.rst

```diff
@@ -31,43 +31,48 @@ for :n:`let @ident := fun {+ @binder} => @term__1 in @term__2`.
    single: ... <: ...
    single: ... <<: ...
 
+.. _type-cast:
+
 Type cast
 ---------
 
 .. insertprodn term_cast term_cast
 
 .. prodn::
-   term_cast ::= @term10 <: @type
+   term_cast ::= @term10 : @type
+   | @term10 <: @type
    | @term10 <<: @type
-   | @term10 : @type
    | @term10 :>
 
 The expression :n:`@term10 : @type` is a type cast expression. It enforces
 the type of :n:`@term10` to be :n:`@type`.
 
-:n:`@term10 <: @type` locally sets up the virtual machine for checking that
-:n:`@term10` has type :n:`@type`.
+:n:`@term10 <: @type` specifies that the virtual machine will be used
+to type check that :n:`@term10` has type :n:`@type` (see :tacn:`vm_compute`).
+
+:n:`@term10 <<: @type` specifies that compilation to OCaml will be used
+to type check that :n:`@term10` has type :n:`@type` (see :tacn:`native_compute`).
 
-:n:`@term10 <<: @type` uses native compilation for checking that :n:`@term10`
-has type :n:`@type`.
+:n:`@term10 :> @type` specifies casting to a base type (for example, an
+underlying inductive type).
```

Comment:
Thanks to your PR #13911, I've discovered that this feature is documented in the refman in this section: https://coq.inria.fr/refman/addendum/program.html#syntactic-control-over-equalities

Instead of this approximate description, I suggest that you just add a link to this section for the documentation of `:>`.

Reply:
Updated. I made it

```
:n:`@term10 :> @type` specifies casting to a base type (for example, an
underlying inductive type).  See :ref:`syntactic_control`.
```

Reply:
This is still unsatisfying. The most problematic thing is this `@type` after `:>` (it doesn't correspond to the grammar). The second issue is that Jason pointed out that this is a Program-specific feature but this description doesn't make this clear at all. I'd suggest something much simpler like:

```
:n:`@term10 :>` casts to the support type in Program-mode. See :ref:`syntactic_control`.
```

It's a bit annoying though that we are losing so much time over documenting a feature that we might very well decide to remove very soon.

Reply:
Updated.

Reply:
Hooray!  It's been a long time coming.  Thanks for your help and patience.

And thanks to @JasonGross for help.